gradle-plugins/plugin-base/src/test/java/io/spine/tools/gradle/given/Given.java

```diff
@@ -30,6 +30,9 @@
     @SuppressWarnings("DuplicateStringLiteralInspection") // Different semantics.
     public static final String JAVA_PLUGIN_ID = "java";
 
+    private Given() {
```

Comment:
I think this is a `...TestEnv` of some kind.

Reply:
I'll move it all into the `GradleProject`, which is a sibling in the same package.